x-pack/plugins/security_solution/public/types.ts

```diff
@@ -176,6 +176,7 @@ export type StartServices = CoreStart &
     telemetry: TelemetryClientStart;
     customDataService: DataPublicPluginStart;
     topValuesPopover: TopValuesPopoverService;
+    timelineFilterManager: FilterManager;
```

Comment:
awesome, smart way to do this that didn't occur to me 👍 can also delete this line https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/store/store.ts#L225 in this pr

Reply:
done